packages/trader/src/Stores/base-store.js

```diff
@@ -31,9 +30,27 @@ export default class BaseStore {
     @observable
     validation_rules = {};
 
+    preSwitchAccountDisposer = null;
```

Comment:
seems like a duplicate of core's `base-store.js`? can all the trader stores be extended from core's base-store instead?

Reply:
@kelly-binary That's the idea and the ultimate goal, however for now I was not certain how to share this class. Any ideas (that don't involve putting it in `deriv-shared`)?

Reply:
@mahdipourziaei-binary ah I see. imho, i think it's ok to put `base-store.js` and `connect.js` in `deriv-shared`, since it's meant to be shared across the packages. but think that can be done later since it'll require more of decoupling work 😃

Reply:
@kelly-binary Ah yes, that's also true. It is possible to move `base-store.js` to `deriv-shared`, yes. I will look into if it can be done cleanly (without any duplication bug), and move it there. The reason it would work is that it's not a singleton, so it should be fine.